src/NumericalAlgorithms/Interpolation/InitializeInterpolationTarget.hpp

```diff
@@ -131,6 +130,15 @@ struct InitializeInterpolationTarget {
                 ::Tags::Variables<typename InterpolationTargetTag::
                                       vars_to_interpolate_to_target>>{}),
         cache);
+    // compute_items_on_target will depend on compute items added in
+    // make_initial_box, so compute_items_on_target must be added
+    // in a separate step.
+    return std::make_tuple(
+        db::create_from<
+            db::RemoveTags<>, db::AddSimpleTags<>,
+            db::AddComputeTags<
```

Comment:
Huh, I had not realized you could pass a list of lists to `AddComputeTags` and it would do the right thing. Neat :) :+1: